dali_tf_plugin/dali_dataset.h

```diff
@@ -75,6 +76,24 @@ class DALIDatasetOp : public tensorflow::data::DatasetOpKernel {
     bool enable_memory_stats;
   };
 
+  // TODO(klecki): I know the name Inputs is not ideal, but we actually get them
+  // in compute from Op inputs.
+  struct Inputs {
+    std::vector<tensorflow::data::DatasetBase *> inputs;
+  };
+
+  // Those are the static Attrs describing inputs
+  struct InputDef {
+    std::vector<std::string> input_names;
+    std::vector<std::string> input_layouts;
+  };
+
+  struct InputDesc : Inputs, InputDef {
```

Comment:
Typically a descriptor describes one entity. InputDescs / InputDefs?

Reply:
Yup, I access them as vectors: 
  OP_REQUIRES_OK(context, context->GetAttr(kInputNames, &def.input_names));

I could maybe repack them.

Reply:
But than I need to repack them back for serialization, I don't it's not just adding more boilerplate around this.